src/components/SwapInfo.vue

```diff
@@ -15,6 +15,14 @@
         <formatted-amount class="price-impact-value" :value="priceImpactFormatted">%</formatted-amount>
       </value-status-wrapper>
     </info-line>
+    <info-line :label="t('swap.route')">
+      <value-status-wrapper>
```

Comment:
`<value-status-wrapper>` component is not needed here. Its purpose is to add color to the wrapper value, depending on the passed numeric value